core/modules/layout/css/layout.admin.css

```diff
@@ -161,7 +161,7 @@
 
 /* Layout dialogs. */
 .layout-dialog {
-  max-width: 600px;
+  max-width: min(95vw, 48em);
```

Comment:
For reference, we started accepting `min()`, `max()`, and `calc()` in Backdrop 1.29.0 as part of https://github.com/backdrop/backdrop-issues/issues/6526, which was a follow-up to Icon API. Though there are a number of `calc()` calls with fallbacks in Basis, which has been in place since 2016.